content/applications/finance/accounting/customer_invoices.rst

```diff
@@ -4,225 +4,221 @@
+  :menuselection:`Accounting --> Customers --> Follow-up Reports`.
+- To get a full overview of the customer's open invoices, click on its line.
+- To open a payment reminder to be sent to the customer, click the :guilabel:`Follow up` button.
+- To exclude any invoices from the follow-up, enable the option :guilabel:`Exclude from Follow-ups`
+  for the selected ones.
 
-Apply the payment. Below, you can see that the payment has been added to
-the invoice.
+.. _accounting/invoices/aging-report:
 
-.. image:: customer_invoices/invoice10.png
+Aged Receivable
+---------------
 
-Payment Followup
-----------------
+To review outstanding customer invoices and their related due dates, use the
+:ref:`Aged Receivable <accounting/reporting/aged-receivable>` report. To access it, go to
+:menuselection:`Accounting --> Reporting --> Aged Receivable`.
 
-There's a growing trend of customers paying bills later and later.
-Therefore, collectors must make every effort to collect money and
-collect it faster.
+.. _accounting/invoice/reporting:
 
-Odoo will help you define your follow-up strategy. To remind customers
-to pay their outstanding invoices, you can define different actions
-depending on how severely overdue the customer is. These actions are
-bundled into follow-up levels that are triggered when the due date of an
-invoice has passed a certain number of days. If there are other overdue
-invoices for the same customer, the actions of the most overdue invoice
-will be executed.
+Reporting
+=========
 
-By going to the customer record and diving into the "Overdue Payments"
-you will see the follow-up message and all overdue invoices.
+Profit and Loss
+---------------
 
-.. image:: customer_invoices/invoice11.png
+The :ref:`Profit and Loss <accounting/reporting/profit-and-loss>` statement shows details of income
+and expenses.
 
-.. image:: customer_invoices/invoice12.png
+Balance sheet
+-------------
 
-.. _customer-invoices/aging-report:
+The :ref:`Balance Sheet <accounting/reporting/balance-sheet>` summarizes the company's liabilities,
+assets, and equity at a specific time.
 
-Customer aging report:
-~~~~~~~~~~~~~~~~~~~~~~
+.. _accounting/invoice/resequencing:
 
-The customer aging report will be an additional key tool for the
-collector to understand the customer credit issues, and to prioritize
-their work.
+Resequencing
+============
 
-Use the aging report to determine which customers are overdue and begin
-your collection efforts.
+:guilabel:`Resequence` is a technical feature used to change the sequence of documents. It can be
+helpful to resequence invoice numbers when importing invoices from another invoicing or accounting
+system, mainly if the reference originates from the previous software and continuity for the current
+year must be maintained without restarting from the beginning.
 
-.. image:: customer_invoices/invoice13.png
+The sequence can be modified once an invoice is in the system.
 
-Profit and loss
----------------
+.. note::
```

Comment:
Great note. I would also add that sequence changes also affect the format of future invoices' sequence.